docs/04-Scripts.md

```diff
@@ -1 +1,4 @@
 # 4.0 - Scripts
+
+
+![pipeline_flow](images/pipeline_flow.png)
```

Comment:
link to the image broken (I think you renamed the files?)

Reply:
fixed!